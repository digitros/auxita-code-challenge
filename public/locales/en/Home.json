{
  "first": "Here you can calculate some information regarding your health state using Blood Pressure data and eGFR to estimate if you have any disease.",
  "second": "Please feel free to select any of the options in the navigation panel to start calculating your health state at any moment.",
  "third": "This platform is developed using React.Js and other tools.",
  "title": "Welcome to healthCalculator"
}
