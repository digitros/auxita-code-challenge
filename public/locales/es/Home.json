{
  "first": "Aquí puede calcular cierta información sobre su estado de salud utilizando los datos de presión arterial y la TFGe para estimar si tiene alguna enfermedad",
  "second": "No dude en seleccionar cualquiera de las opciones en el panel de navegación para comenzar a calcular su estado de salud en cualquier momento.",
  "third": "Esta plataforma se desarrolla utilizando React.Js y otras herramientas.",
  "title": "Bienvenido a healthCalculator"
}
