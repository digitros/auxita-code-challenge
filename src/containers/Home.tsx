import React from 'react';

const Home = () => <h1>Auxita</h1>;

export default Home;
