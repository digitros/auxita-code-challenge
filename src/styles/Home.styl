main
    display: flex
    justify-content: center
    align-items: center
    width: calc(100vw - 250px)
    height: calc(100vh - 100px)
    background: #F0F2F8

.home
    margin: 10px
    text-align: center