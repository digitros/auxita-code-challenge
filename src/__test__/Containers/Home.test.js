import React from 'react';
import {create} from 'react-test-renderer'

/* Containers */
import Home from '../../containers/Home';

jest.mock('../../i18n.js');

describe('<Home />', () => {
  test('Home snapshot', () => {
    const home = create(<Home />);
    expect(home.toJSON()).toMatchSnapshot();
  });
});
