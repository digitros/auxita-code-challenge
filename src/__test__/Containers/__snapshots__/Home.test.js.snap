// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Home snapshot 1`] = `
<main>
  <section
    className="box box-medium"
  >
    <div
      className="home"
    >
      <h1>
        Home:title
      </h1>
      <p>
        Home:first
      </p>
      <p>
        Home:second
      </p>
      Home:third
      <p />
    </div>
  </section>
</main>
`;
